### PR TITLE
Done 

### PR DESCRIPTION
**Pull request by ranelbe**
Done

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # The GPT-Explainer Project
 
+To run the script type:
```
Comment (excellenteam-scaleup) [inline README.md]: Nicely explained

File: api_manager.py
Comment (excellenteam-scaleup) [inline api_manager.py:1]: Good logical separation of the API to a module.
Not sure we need a class for managing it, but it's a possible way for it

File: api_manager.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class APIManager:
+    """ A class to manage the API calls to the OpenAI api. """
+
+    def __init__(self):
+        """ initialization.
+        Set the API key from an environment variable.
+        """
+        openai.api_key = os.getenv("OPENAI_API_KEY")  # an environment variable
```
Comment (excellenteam-scaleup) [inline api_manager.py]: It would be better for the environment variable to be in a config file, or at least a constant and not magically appear here

File: main.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import asyncio
+import argparse
+from pptx_summarizer import PPTXSummarizer
+
+
+def cli_configuration():
```
Comment (excellenteam-scaleup) [inline main.py]: Nice implement of bonus requierments

File: pptx_summarizer.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import json
+
+from pptx_parser import PPTXParser
+from prompt_composer import PromptComposer
+from api_manager import APIManager
+import asyncio
+
+OUTPUT_FILE_NAME = 'result.json'
```
Comment (excellenteam-scaleup) [inline pptx_summarizer.py]: One of the requirements was for the json file to have the same name like the original pptx file

File: pptx_parser.py
Comment (excellenteam-scaleup) [inline pptx_parser.py:1]: Again - good separation of logical parts

File: prompt_composer.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
+
+PROMPT_SUFFIX = """provide a detailed explanation for this presentation slide,
```
Comment (excellenteam-scaleup) [inline prompt_composer.py]: Good prompt

File: requirements.txt
Comment (excellenteam-scaleup) [inline requirements.txt:1]: Nice

File: test.pptx
Comment (excellenteam-scaleup) [inline test.pptx:1]: It's nice that you added a test file. It would be nice if it also had some edge-cases like an empty slide

**Review by excellenteam-scaleup — APPROVED** [review]
Very good job!
Your git work looks great:
Commit messages are a little short, but explain the commit
Good logical division of branches, division to modules and files.
Your code is working, documented, follows PEP conventions, and divided into small simple functions.
note that when multiple pptx files are given the output needs to be saved in separate json files to keep track.
 
Keep up the good work!
